### PR TITLE
RAC-4819 - convert decommission nodes script to FIT

### PR DESCRIPTION
**Pull request by larry-dean**
* Created decommission validation workflow

### CODE REVIEW

**Comment by larry-dean** [comment]
@hohene @johren @BrentHiggins

**Comment by BrentHiggins** [comment]
+1

File: lib/graphs/bootstrap-decommission-node-test-graph.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+// Copyright 2018, EMC, Inc.
```
Comment (johren) [inline lib/graphs/bootstrap-decommission-node-test-graph.js]: Copyright (c) 2017 Dell Inc. or its subsidiaries. All Rights Reserved

**Comment by larry-dean** [comment]
Pushed up changes suggested by PR review

**Pull request merged**